docs/user-guide/at-tls-configuration.md

```diff
@@ -1,23 +1,676 @@
 
-:::info Required roles: security administrator
+:::info Role: security administrator
 :::
 
-## Configuration Parameters
+- [AT-TLS configuration for Zowe](#at-tls-configuration-for-zowe)
+  - [Limitations](#limitations)
+- [AT-TLS rules](#at-tls-rules)
+  - [Inbound rules](#inbound-rules)
+  - [Outbound rules](#outbound-rules)
+    - [For z/OSMF](#for-zosmf)
+    - [For communication between API Gateway and other core services](#for-communication-between-api-gateway-and-other-core-services)
+    - [For communication between API Gateway and southbound services](#for-communication-between-api-gateway-and-southbound-services)
+    - [Services that validate tokens against the API Mediation Layer](#services-that-validate-tokens-against-the-api-mediation-layer)
+  - [Ciphers](#ciphers)
+- [Using AT-TLS for API ML in High Availability](#using-at-tls-for-api-ml-in-high-availability)
+- [Multi-tenancy deployment](#multi-tenancy-deployment)
+- [AT-TLS Troubleshooting](#at-tls-troubleshooting)
+  - [The message `This combination of port requires SSL` is thrown when accesing an API ML service through a Browser](#the-message-this-combination-of-port-requires-ssl-is-thrown-when-accesing-an-api-ml-service-through-a-browser)
+  - [AT-TLS rules are not applied](#at-tls-rules-are-not-applied)
+  - [Non matching ciphers / protocols](#non-matching-ciphers--protocols)
+  - [Additional troubleshooting](#additional-troubleshooting)
+- [Full example of AT-TLS configuration](#full-example-of-at-tls-configuration)
+- [Additional Zowe feature configuration with AT-TLS](#additional-zowe-feature-configuration-with-at-tls)
+
+## AT-TLS configuration for Zowe
+
+Follow these steps to configure Zowe to support AT-TLS:
 
-To enable AT-TLS for Zowe components, configure the following parameters:
 ```yaml
 zowe:
     network:
+        # For inbound traffic rules:
         server:
             tls:
                 attls: true
+        # If outbound traffic rules will be configured:
+        client:
+          tls:
+            attls: true
 ```
 
-## Component-Specific Configuration
+While the Zowe Server components do not handle TLS on its own with AT-TLS enabled, the API Mediation Layer (API ML) requires information about the server certificate that is defined in the AT-TLS rule. Ensure that the server certificates provided by the AT-TLS layer are trusted in the configured Zowe keyring. We strongly recommend that AT-TLS be configured with the same Zowe keyring.
+
+:::note Notes
+
+- As the API ML Gateway is a core component of API ML, other components that need to interact with the Gateway, such as Zowe ZLUX App Server, also require AT-TLS configuration.
+
+- Do not set `attls: true` together with `minTls` or `maxTls`. Zowe does not handle TLS in AT-TLS aware mode.
```

Comment:
Really?

Reply:
It's a schema validation, perhaps it's too strict, especially if there are going to be mixed scenarios with AT-TLS and also API ML handling TLS, for routing for example. cc @1000TurquoisePogs

Reply:
Will update after schema validation